src/CommentList.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Comment from "./Comment.js";
+
+function CommentList(props) {
```

Comment:
this function has a mistake and it doesn't work

Reply:
fixed